library/src/main/java/com/novoda/simplechromecustomtabs/SimpleChromeCustomTabs.java

```diff
@@ -20,42 +19,35 @@
 
 public final class SimpleChromeCustomTabs implements WebNavigator, Connection, AvailableAppProvider {
 
-    private static Context applicationContext;
     private Connection connection;
     private WebNavigator webNavigator;
     private AvailableAppProvider availableAppProvider;
 
-    private SimpleChromeCustomTabs() {
-        //no-op
-    }
-
-    private static class LazyHolder {
-        private static final SimpleChromeCustomTabs INSTANCE = new SimpleChromeCustomTabs();
+    /**
+     * @deprecated There is no need to call it anymore in order to use this library,
```

Comment:
I'm pro keeping the `deprecated` after thinking on the impact of this breaking change.

- The release is going to be documented to the clients can choose if they can/want to upgrade or not
- Effort in order to remove/include this `initialise` is minimal in case this needs to be added in the future.

I cannot see a reason why we would need to add the initialisation in the future, but this breaking change in my opinion has a minimal impact.